crates/trigger/src/cli.rs

```diff
@@ -119,15 +120,15 @@ where
 
         // TODO: I assume there is a way to do this with a single let mut loader: Box<dyn Loader>
         // variable instead of the entire executor.
-        let executor: Executor = match self.oci {
+        let executor: Executor = match self.from_registry {
             true => {
                 let loader =
                     OciTriggerLoader::new(working_dir, self.allow_transient_write, None).await?;
 
                 let trigger_config =
                     TriggerExecutorBuilderConfig::load_from_file(self.runtime_config_file.clone())?;
 
-                let _sloth_warning = warn_if_wasm_build_slothful();
+                _ = warn_if_wasm_build_slothful();
```

Comment:
I don't think this is safe.  If I recall correctly, the reason the variable is named is because we found that discarding it caused the drop to happen immediately.

Reply:
Even if it is safe, I think giving it a `_name` helps clarify its purpose in scoping.